plugins/spu2-x/src/PS2E-spu2.cpp

```diff
@@ -245,21 +245,13 @@ SPU2dmaInterrupt(s32 channel)
         return Cores[1].NewDmaInterrupt();
 }
```

Comment:
I think SPU2dmaRead, SPU2dmaWrite, and SPU2dmaInterrupt (just above) needs to be removed as well?